pkg/cedar/core/system_summary.go

```diff
@@ -132,15 +137,28 @@ func (c *Client) GetSystem(ctx context.Context, systemID string) (*models.CedarS
 
 type systemSummaryParamFilterOpt func(*apisystems.SystemSummaryFindListParams)
 
+type systemSummaryParamOpts struct{}
+
+// SystemSummaryParams contains methods for options to pass to system summary calls
+var SystemSummaryParams = systemSummaryParamOpts{}
+
+// WithDeactivatedSystems returns all systems
+func (_ systemSummaryParamOpts) WithDeactivatedSystems() systemSummaryParamFilterOpt {
```

Comment:
you can remove the var all together
```suggestion
func (systemSummaryParamOpts) WithDeactivatedSystems() systemSummaryParamFilterOpt {
```

Reply:
TIL, thank you!

Reply:
Added in [6397f88](https://github.com/CMSgov/easi-app/pull/2605/commits/6397f8829d989672510ee51d085b7f114cf1781c)